src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1283,7 +1288,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
         End Function
 
         Private Function CreateBoundRaiseEventStatementOperation(boundRaiseEventStatement As BoundRaiseEventStatement) As IExpressionStatement
-            Dim expression As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundRaiseEventStatement.EventInvocation))
+            Dim expression As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() GetRaiseEventExpression(boundRaiseEventStatement))
```

Comment:
>Dim expression As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() GetRaiseEventExpression(boundRaiseEventStatement)) [](start = 12, length = 125)

Can we get in a situation here that this node and the expression statement node, created below, both refer to the same syntax and  both are marked as explicit? Perhaps the expression statement node should always be marked as implicit? Or should we have a node that represents the raise event statement instead?

Reply:
I think changing this to IRaiseEventStatement makes sense.

---
In reply to: [140925916](https://github.com/dotnet/roslyn/pull/22243#discussion_r140925916) [](ancestors = 140925916)